jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -28,23 +28,30 @@
     "vuex": "^3.6.2"
   },
   "devDependencies": {
+    "@babel/register": "^7.16.9",
+    "@istanbuljs/nyc-config-babel": "^3.0.0",
     "@vue/cli-plugin-babel": "^4.5.15",
     "@vue/cli-plugin-e2e-cypress": "^4.5.15",
     "@vue/cli-plugin-eslint": "^4.5.15",
-    "@vue/cli-plugin-unit-mocha": "^4.5.15",
+    "@vue/cli-plugin-unit-mocha": "~4.5.0",
     "@vue/cli-service": "^4.5.15",
     "@vue/eslint-config-standard": "^6.1.0",
     "@vue/test-utils": "^1.3.0",
     "babel-eslint": "^10.1.0",
+    "babel-plugin-istanbul": "^6.1.1",
     "chai": "^4.3.4",
     "eslint": "^7.22.0",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-standard": "^4.1.0",
     "eslint-plugin-vue": "^8.2.0",
+    "istanbul-instrumenter-loader": "^3.0.1",
+    "nyc": "^15.1.0",
```

Comment:
The pair istanbul (instrumenter) and nyc (coverage reporter) are similar to Jacoco or Cobertura.